packages/link/react.d.ts

```diff
@@ -0,0 +1,10 @@
+/// <reference types="react" />
+
+export type Appearance = 'default' | 'subtle'
+
+export interface LinkProps {
```

Comment:
Is the *Props name a convention?

Reply:
That's the way I've seen it everywhere else

Reply:
Did a little more research, just to make sure that `*Props` wasn't a homegrown convention. Every place I researched that gives examples of React + Typescript are using this naming convention.